common/k8s.go

```diff
@@ -0,0 +1,30 @@
+package common
+
+import (
+	"github.com/pkg/errors"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+// RESTConfig returns the rest config needed by most k8s-related clients.
+// Call this **after** flag.Parse().
+func RESTConfig(kubeconfig string) (*rest.Config, error) {
+	var (
+		err    error
+		config *rest.Config
+	)
+
+	if kubeconfig == "" {
+		config, err = rest.InClusterConfig()
+		if err != nil {
+			err = errors.Wrapf(err, "error loading the config inside the cluster")
+		}
```

Comment:
This isn't really much of a problem (and won't come up at all unless we end up running this utility locally), but I think that this would always require us to use the `--kubeconfig` command-line option rather than relying on the `KUBECONFIG` environment variable or using the default configuration file (`$HOME/.kube/config) if the command-line option isn't used. If we'll only be running this utility on the command line rarely then that's not a big deal, but I thought I'd mention it just in case.